openproblems/tasks/spatial_decomposition/methods/destvi.py

```diff
@@ -1,5 +1,6 @@
 from ....tools.decorators import method
 from ....tools.utils import check_version
+from .._utils import split_sc_and_sp
```

Comment:
If we're importing methods from `_utils`, it shouldn't be a private submodule -- rename to `utils`